lib/ash_json_api/api/verifiers/verify_open_api_grouping.ex

```diff
@@ -0,0 +1,24 @@
+defmodule AshJsonApi.Api.Verifiers.VerifyOpenApiGrouping do
+  @moduledoc """
+  Ensures that tag is present if group_by is :api
+  """
+  use Spark.Dsl.Verifier
+
+  def verify(dsl) do
+    tag = AshJsonApi.Api.Info.tag(dsl)
+    group_by = AshJsonApi.Api.Info.group_by(dsl)
+
+    unless group_by === :api && tag !== "" do
+      {:error,
```

Comment:
You aren't returning the error here. You should raise this error instead of doing `{:error, ...}`